src/services/datahub-client.doubles.ts

```diff
@@ -173,9 +177,19 @@ export class HardcodedDatahubClient implements DatahubClient {
       withEmail: x.withEmail,
     }));
 
+    let pagesSubArray = [];
+
+    if (pageSize) {
+      const indexStart = pageSize * pageNumber - 1;
```

Comment:
Is it fine? If not, could you add a test to take into account this?

Reply:
it's only for local purposes the real BE it's no like this

Reply:
Oh, I understand, but I have two observations:

* Please verify this line, because, if I am not wrong it should be:

    ```suggestion
          const indexStart = pageSize * pageNumber - 1;
    ```

* I have checked [the related demo page](https://cdn.fromdoppler.com/doppler-webapp/demo-build1658/#/reports), and I cannot see anything related to the pagination. Maybe is this code not necessary?

Reply:
Thanks i changed the parenthesis